index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Apple</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.9.0/css/all.css"
+    integrity="sha384-i1LQnF23gykqWXg6jxC2ZbCbUMxyw5gLZY6UiUS98LYV5unm8GWmfkIS6jqJfb4E"
+    crossorigin="anonymous" />
+    <link rel="stylesheet" href="assets/css/style.css">
+</head>
+<body>
+
+    <main class="site__header main">
+        <nav class="nav">
+            <ul class="nav__list">
+                <li class ="nav-logo"><a href=""><img src="assets/img/5d34bc9073b21.png" alt=""></a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Mac</a></li>
+                <li><a href="#">iPod</a></li>
+                <li><a href="#">iPhone</a></li>
+                <li><a href="#">iPad</a></li>
+                <li><a href="#">iTunes</a></li>
+                <li class="nav-last"><a href="#">Support</a></li>
+                <li>
+                    <div class=search-bar>
+                            <!-- <i class="fas fa-search"></i> -->
+                            <input type="text">
+                    </div>
+                </li>
+            </ul>
+        </nav>
+
+        <section class="header">
+            <h1><a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_title_medium.png" alt=""></a></h1>
+            <h2><a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/your_verse_subhead_medium.png" alt=""></a></h2>
+            <div class="header__verse">
+                <a href="#">
+                    <img src="https://web.archive.org/web/20140301004341im_/http://images.apple.com/home/images/cta_your_verse_27pt.png" alt="">
+                </a>
+            </div>
+        </section>
+
+    </main>
+
+    <aside>
+        <ul class="aside-container">
+            <li class="aside__box"><a href="#"><img src="https://web.archive.org/web/20140301004349im_/http://images.apple.com/home/images/promo_iphone5s.jpg" alt=""></a></li>
+            <li class="aside__box"><a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" alt=""></a></li>
+            <li class="aside__box"><a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years.jpg" alt=""></a></li>
+            <li class="aside__box"><a href="#"><img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg" alt=""></a></li>
+        </ul>
```

Comment:
`REQUIRED`

Use CSS background property to set all the list item content as background instead of using a `<img>` tag.